src/routes/safe/components/AddressBook/ImportEntryModal/index.tsx

```diff
@@ -92,16 +91,19 @@ const ImportEntryModal = ({ importEntryModalHandler, isOpen, onClose }) => {
   const validateCsvData = (data) => {
```

Comment:
can we extract the validation functions as utils, type them and add tests?

Reply:
I'll create an issue to address this